README.md

```diff
@@ -10,49 +10,20 @@ We decided to take notes that create a reproducible experience. We document our
-- a basic understanding of [IPv4 and subnets](https://www.linkedin.com/learning/cisco-ccna-200-301-cert-prep-1-network-fundamentals-and-access/ipv4-addressing-and-subnetting)
-- optional: understand how [`jq` works](https://stedolan.github.io/jq/tutorial/) (if you feel motivated)
+- a basic understanding of [IPv4 and subnets](./chapters/appendix/prefixes-and-subnet-masks.md)
+- optional: a basic undestanding of docker [Course link](https://www.linkedin.com/learning/learning-docker-2018/why-create-containers-using-docker)
 - optional for the super motivated: understand the [communication protocol layers](https://datatracker.ietf.org/doc/rfc1122/) we'll be working with
-- get some software installed:
-  - [colima](https://smallsharpsoftwaretools.com/tutorials/use-colima-to-run-docker-containers-on-macos/) or [docker desktop](https://www.docker.com/products/docker-desktop/) for mac™
-  - `jq` by whatever means necessary (e.g. `brew install jq`)
-
-## DISCLAIMERS
-
-We've put some effort into explaining the commands that we're using. However, if we use a command with a flag that doesn't have explanation, check the command's `help` for more details; e.g. `ping --help`. Alternatively, check the command's manpage; e.g. `man ping`.
-
-## General thoughts
-
-What if we build a little internet?
-
-### Intentions
 
-- Documenting our process for our future friends
-- How to build the internet blog
+### Software
 
-### What concepts are involved
+You will need to install [colima](https://smallsharpsoftwaretools.com/tutorials/use-colima-to-run-docker-containers-on-macos/) or [docker desktop](https://www.docker.com/products/docker-desktop/) to manage your docker instances. We go into more detail what these software are and how to get started with them in [our prequel chapter 000](./chapters/000-prequel-how-does-this-work/README.md).
 
-- Routing/Networking basics
-- Layer-2 concepts
-- BGP specific routing concepts
-- DNS root / federation
-- Docker stuffs
-- TLS fundamentals
-- HTTP fundamentals
-- VPN: TCP / UDP differences - maximum segment-size discovery
-
-### What tools should we use?
-
-- What are we doing today ?
-- Github projects (Kanban)
-
-## How to get started reading and getting involved
+## DISCLAIMERS
 
-- Start with [Chapter 1](https://github.com/psbanka/build-your-own-internet/blob/main/chapters/001-getting-started/README.md).
-- If you want to make changes to this repository (e.g. fixing errors or adding sections), fork this repository and submit a pull-request
-- If you have a general question and want someone to talk to, [submit an issue](https://github.com/psbanka/build-your-own-internet/issues/new)
+We've put some effort into [explaining the commands]() that we're using. However, if we use a command with a flag that doesn't have explanation, check the command's `help` for more details; e.g. `ping --help`. Alternatively, check the command's manpage; e.g. `man ping`.
```

Comment:
Merge after https://github.com/psbanka/build-your-own-internet/pull/26 and add link to the commands document.